pyriemann/datasets.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from sklearn.utils.validation import check_random_state
+from .sampling import generate_random_spd_matrix, sample_gaussian_spd
+
+
+def make_gaussian_blobs(n_samples=100, n_dim=2, class_sep=1.0, class_disp=1.0,
+                        random_state=42):
```

Comment:
```suggestion
                        random_state=None):
```
don't make the function deterministic by default